irispy/obsid.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pandas as pd
+from astropy import units as u
+from pkg_resources import resource_filename
+
+
+class Obsid(dict):
```

Comment:
Do you think the class should be renamed ```OBSID```?  If not, it should probably be at least ```ObsId``` or ```ObsID``` since classes should be camel-case.